release notes/upcoming.md

```diff
@@ -2,11 +2,10 @@ TODO: Intro
 
 ## New Features!
 
-### Category: Title (#533)
+### HTTP: request body compression (#988)
 
-Description of feature.
+Now all http methods have an additional param called `compression`, with a single possible value `gzip`, which if set will send the body compressed as gzip. It will also correctly set both `Content-Encoding` and `Content-Length`.
```

Comment:
Probably some extra notes here at the end would need to be added, like:
> It will also correctly set both `Content-Encoding` and `Content-Length` header values, if they haven't explicitly been specified.